sovrin_client/test/cli/helper.py

```diff
@@ -191,3 +198,45 @@ def new():
             with Looper(debug=False) as looper:
                 yield new()
     return _
+
+
+# marker class for regex pattern
+class P(str):
+    def match(self, other):
+        return re.match('^{}$'.format(self), other)
+
+
+def check_wallet(cli,
+                 totalLinks=None,
+                 totalAvailableClaims=None,
+                 totalSchemas=None,
+                 totalClaimsRcvd=None,
+                 within=None):
+    async def check():
+        assert (totalLinks is None or
+                totalLinks == len(cli.activeWallet._links))
+
+        tac = 0
+        for li in cli.activeWallet._links.values():
+            tac += len(li.availableClaims)
+
+        assert (totalAvailableClaims is None or
+                totalAvailableClaims == tac)
+
+        assert (totalSchemas is None or
+                totalSchemas == len(await cli.agent.prover.wallet.getAllSchemas()))
+
+        assert (totalClaimsRcvd is None or
+                totalClaimsRcvd == len((await cli.agent.prover.wallet.getAllClaims()).keys()))
+
+    if within:
+        cli.looper.run(eventually(check, timeout=within))
+    else:
+        check()
+
+
+def wallet_starting_state(totalLinks=0,
```

Comment:
Why do we pass arguments if we dont use them? Is the `return locals()` a temporary thing?